lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -834,11 +832,11 @@ class DropDown {
 
   /**
    * @param {Array<Node>} allNodes
-   * @param {?Node=} startNode
-   * @returns {Node}
+   * @param {?HTMLElement=} startNode
+   * @returns {HTMLElement}
    */
   _getNextSelectableNode(allNodes, startNode) {
-    const nodes = allNodes.filter((node) => {
+    const nodes = allNodes.filter(/** @return {node is HTMLElement} */ (node) => {
```

Comment:
also not `querySelector` related, but this was already checking `instanceof HTMLElement`, so it seems good